### PR TITLE
[Merged by Bors] - [CU-2jp2wkt][CU-2jpxe6][CU-2jp2xng] docs(book/products): Add new pages to book

### PR DESCRIPTION
**Pull request by Martin-Kiss**
## Issue
https://app.clickup.com/t/2jp2wkt
https://app.clickup.com/t/2jp2xe6
https://app.clickup.com/t/2jp2xng

## Description
This PR tries a different workflow as previously multiple PR's affecting the `SUMMARY.md` led to issues. Unifying these changes to the summary is expected to result in less merge conflicts.

## Checklist

- [X] | Added Apollo
- [X] | Added Pablo
- [X] | Added CHAOS
- [X] | Minor Fixes
- [X] I have updated the `SUMMARY.md` to reflect changes made by this PR

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2jp2xng Add CHAOS to mdbook](https://app.clickup.com/t/2jp2xng)

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2jp2wkt Add Apollo to Mdbook](https://app.clickup.com/t/2jp2wkt)

**Comment by JafarAz** [comment]
@Martin-Kiss Everything looks good, except can you position Apollo below Mosaic, not above on the sidebar. And also, add a line underneath Apollo's title on the main Apollo page.

**Review by 0xBrainjar2 — APPROVED** [review]
left 2 comments for @JafarAz then LGTM

File: book/src/products/pablo/xPBLO-fNFT-staking.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+---
+
+Users are able to stake their PBLO native token in order to earn yield. 20% of the total trading fees on Pablo will be distributed to xPBLO stakers. In addition to trading fees, xPBLO holders will receive rewards in the form of additional PBLO and other incentives.
+
+Users can stake PBLO by locking their tokens for different durations. In return, they are given xPBLO fNFTs, which serve as composable representatives of the value of their asset which can then be used for different purposes across DeFi ecosystems. Users can then burn their xPBLO tokens at the end of their lock period in order to receive their original assets.
+
+
+![pablo_architecture](./pablo-architecture.png)
+_High level overview of Pablo _
```
Comment (JafarAz) [inline book/src/products/pablo/xPBLO-fNFT-staking.md:11]: this doesn't show as a caption on codespaces
Screenshot 2022-07-08 at 09 26 38
.

**Review by JafarAz — APPROVED** [review]

**Review by 0xBrainjar2 — APPROVED** [review]

**Review by 0xBrainjar2 — APPROVED** [review]

**Review by Dom-Roth — APPROVED** [review]

**Pull request closed without merging**